Resources/config/form-type.xml

```diff
@@ -9,5 +9,13 @@
             <tag name="form.type" alias="cmf_core_checkbox_url_label" />
             <argument type="service" id="router" />
         </service>
+
+        <service id="cmf_core.form.extension.publishable" abstract="true" class="Symfony\Cmf\Bundle\CoreBundle\Form\Extension\PublishableExtension">
+            <argument></argument><!-- Must be overrided when using this service as parent -->
```

Comment:
ah, i think i get this one now. we need to explicitly state what form types we extend with this extension? i like this approach to things. the comment should say what the argument is. or you could completely omit the `<argument>` part and only have the comment.

this makes me think that we could do a compiler pass for a tag on form types that we tag with form_extension and a parameter that says e.g. publishable. the pass could then copy this service and set the parameter to the tagged service. does nothing in that direction already exist? anyways, this was just a brainstorm, we can also do that in a separate pull request if it makes sense at all.